rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.get_alias/10_basic.yml

```diff
@@ -16,15 +19,44 @@ setup:
             test_alias: {}
             test_blias: {}
 
+  - do:
+      security.put_role:
+        name: "get_alias_test_role"
+        body: >
+          {
+            "cluster": [],
+            "indices": [
+              {
+                "names": ["test*", "myindex", "non-existent", "another-non-existent", "foo"],
```

Comment:
Granting only necessary privileges. Including missing ones here in order to keep the same tests that expect 404 instead of 403.